src/Analysis.cpp

```diff
@@ -346,6 +394,10 @@ void CalledMethodsAnalysis::doAnalysis(Function &F,
     doAnnos = true;
   }
 
+  if (fnName != "main") {
```

Comment:
Am I correct in thinking that this early return means that the analysis won't run at all on methods not named "main"? If so, that's going to be a problem (probably sooner rather than later).

What's the reason for adding this?

Reply:
That's correct; I have it here because I didn't need the entire analysis running on every single method as the test cases only focus on the main method. Moving on though, this line will be removed